app/portainer/components/forms/registry-form-gitlab/registry-form-gitlab.html

```diff
@@ -54,7 +54,7 @@
         ng-if="!$ctrl.state.overrideConfiguration"
         ng-click="$ctrl.state.overrideConfiguration = true;"
       >
-        <pr-icon icon="'tool'"></pr-icon>
+        <pr-icon icon="'wrench'"></pr-icon>
```

Comment:
Not sure if intentional, but this change is not present in the EE pr.

Reply:
updated EE